# HG changeset patch
# User Maxim Dounin <mdounin@mdounin.ru>
# Date 1297776703 -10800
# Node ID d420362cacbf42b516127aa8a55afd13f0fb6d5d
# Parent  3970edd4440a49d37253b231b97c72f0b0728eed
Fastcgi: fix large stderr handling without cache.

diff --git a/src/http/modules/ngx_http_fastcgi_module.c b/src/http/modules/ngx_http_fastcgi_module.c
--- a/src/http/modules/ngx_http_fastcgi_module.c
+++ b/src/http/modules/ngx_http_fastcgi_module.c
@@ -1280,6 +1280,8 @@ ngx_http_fastcgi_process_header(ngx_http
                         } else {
                             u->buffer.pos = u->buffer.start;
                         }
+#else
+                        u->buffer.pos = u->buffer.start;
 #endif
 
                         u->buffer.last = u->buffer.pos;
